src/shared/components/error/Error.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import SlideDown from 'react-slidedown';
+import { Svg, Button, TextButton, WarningIcon } from '@nomios/web-uikit';
+import styles from './Error.css';
+
+const fadingLogoSvg = import(/* webpackChunkName: "svg-illustrations-sprite" */ '../../media/illustrations/brand-fading-symbol.svg');
+
+class Error extends Component {
+    state = {
+        showingDetails: false,
+    };
+
+    render() {
+        const { className, text, error, onRetry } = this.props;
+        const { showingDetails } = this.state;
+
+        return (
+            <div className={ classNames(styles.errorScreen, className) }>
```

Comment:
Change the class name to `error`. Beware that the error class is already being used below, perhaps rename that one to `errorBlock`.

Reply:
👍